packages/devtools/lib/src/settings/settings_screen.dart

```diff
@@ -42,31 +50,43 @@ class SettingsScreen extends Screen {
     });
   }
 
+  void _rebuild() {
+    _screenDiv
+      ..clear()
+      ..add(_container..clear());
+    if (serviceManager.hasConnection) {
+      final flutterVersion = div(c: 'flutter-version-container')
+        ..flex(2)
+        ..add(<CoreElement>[
+          CoreElement('h2', text: 'Flutter SDK Version: '),
+          span(text: serviceManager.sdkVersion),
+        ]);
+      _flagList = div(c: 'flag-list-container')
+        ..layoutVertical()
+        ..add(
+          CoreElement('h2', text: 'Dart VM Flag List'),
+        );
+      _container.add(flutterVersion);
+      _container.add(_flagList);
+      _displayFlagList();
+    } else {
+      _screenDiv.add(h2(text: 'Settings'));
+      _screenDiv.add(span(text: 'TODO'));
```

Comment:
instead of TODO, I would leave this blank when the vm hasn't loaded yet.

Reply:
updated